Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN wget https://github.com/jwilder/docker-gen/releases/download/$DOCKER_GEN_VER
 
 COPY network_internal.conf /etc/nginx/
 
-COPY . /app/
+COPY app /app/
```

Comment:
> Adding the trailing `/.` means "copy the _content_ of the directory to ", whereas _without_, it means "Copy to . Depending on if `<target>` exists or not, and wether or not `<target>` is a directory, this can cause `<source>` to either land in (in this case) `/app/` or `/app/app/`. See https://docs.docker.com/engine/reference/commandline/cp/#extended-description.

This is not true about the Dockerfile `COPY` command. See [1]:
> If `<src>` is a directory, the entire contents of the directory are copied, including filesystem metadata.
    Note
    The directory itself is not copied, just its contents.

[1] https://docs.docker.com/engine/reference/builder/#copy

Reply:
> I guess we need to make the same changes in `Dockerfile.alpine` as well?

Sure.